flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -535,20 +540,16 @@ private void startTriggeringCheckpoint(CheckpointTriggerRequest request) {
 							request.getOnCompletionFuture()),
 						timer);
 
-			final CompletableFuture<?> masterStatesComplete = pendingCheckpointCompletableFuture
-					.thenCompose(this::snapshotMasterState);
+			final CompletableFuture<?> masterStatesComplete = pendingCheckpointCompletableFuture.thenCompose(this::snapshotMasterState);
 
-			final CompletableFuture<?> coordinatorCheckpointsComplete = pendingCheckpointCompletableFuture
-					.thenComposeAsync((pendingCheckpoint) ->
-							OperatorCoordinatorCheckpoints.triggerAndAcknowledgeAllCoordinatorCheckpointsWithCompletion(
-									coordinatorsToCheckpoint, pendingCheckpoint, timer),
-							timer);
+			final CompletableFuture<?> coordinatorCheckpointsComplete = pendingCheckpointCompletableFuture.thenComposeAsync(
+					pendingCheckpoint -> triggerAndAcknowledgeAllCoordinatorCheckpointsWithCompletion(coordinatorsToCheckpoint, pendingCheckpoint, timer),
+					timer);
```

Comment:
Those lines should be wrapped somehow, also in other places after your changes lines are too long.

Reply:
As Flink guidelines don't name any specific limit, I usually assume that if it fits on my 14" screen then it could fit on most other devices:) 
I think it makes sense to split the line to attract attention to something important, but here it's just one lambda with one method call.

Reply:
Hmmm, indeed it doesn't say anything. At least not for java (for Scala it says 100).

Those lines are to wide for example for side-by-side comparisons (diffs) or when opening two files with vertical split.

Reply:
I think the code is more often read as it is, without comparison (and for comparison of long lines I use line-by-line format).
Please let me know if you prefer it to be wrapped anyways.